content/en/docs/collector/internal-telemetry.md

```diff
@@ -260,6 +260,57 @@ categorized by instrumentation type.
 | `rpc_server_response_size`        | Measures the size of RPC response messages (uncompressed).                                | Histogram |
 | `rpc_server_responses_per_rpc`    | Measures the number of messages sent per RPC. Should be 1 for all non-streaming RPCs.     | Histogram |
 
+### Telemetry maturity levels
+
+#### Traces
+
+Tracing instrumentation is still under active development, and changes might be made to span names, attached attributes, instrumented endpoints, or other aspects of the telemetry. Until this feature graduates to stable, there are no guarantees of backwards compatibility for tracing instrumentation.
+
+#### Metrics
+
+The Collector's metrics follow a four-stage lifecycle:
+
+> Alpha metric → Stable metric → Deprecated metric → Deleted metric
+
+##### Alpha
+
+Alpha metrics have no stability guarantees. These metrics can be modified or deleted at any time.
+
+##### Stable
+
+Stable metrics are guaranteed to not change. This means:
+
+- A stable metric without a deprecated signature will not be deleted or renamed.
+- A stable metric's type and attributes will not be modified.
+
+##### Deprecated
+
+Deprecated metrics are slated for deletion but are still available for use. These metrics include an annotation about the version in which they became deprecated. For example:
+
+Before deprecation:
+
+```sh
+# HELP otelcol_exporter_queue_size this counts things
```

Comment:
Do we only consider Prometheus metrics or is there also a convention we want if deprecating an Otel metric?

Reply:
I'd expect the description of the metric to include a note about deprecation. The description will be rendered differently based on the exporter configured